tests/integration/components/bs-form-test.js

```diff
@@ -97,6 +97,17 @@ module('Integration | Component | bs-form', function(hooks) {
     assert.ok(submit.called, 'onSubmit action has been called');
   });
 
+  test('Clicking a normal button that triggers the yielded submit action submits the form', async function(assert) {
+    let submit = this.spy();
+    this.actions.submit = submit;
+    await render(
+      hbs`{{#bs-form onSubmit=(action "submit") as |form|}}{{#bs-button onClick=form.submit}}Submit{{/bs-button}}{{/bs-form}}`
+    );
+
+    await click('button');
+    assert.ok(submit.called, 'onSubmit action has been called');
+  });
+
```

Comment:
Just realized I didn't add a test of the `form.validate` here, will put that in as part of the next pass